guides/client/js-interop.md

```diff
@@ -253,10 +253,15 @@ and the return value is ignored.
 For example, the following option could be used to guarantee that some attributes set on the client-side are kept intact:
 
 ```javascript
-onBeforeElUpdated(from, to){
-  for (const attr of from.attributes){
-    if (attr.name.startsWith("data-js-")){
-      to.setAttribute(attr.name, attr.value);
+...
+let liveSocket = new LiveSocket("/live", Socket, {
+  params: { _csrf_token: csrfToken },
+  hooks: Hooks,
+  onBeforeElUpdated(from, to) {
+    for (const attr of from.attributes) {
+      if (attr.name.startsWith("data-js-")) {
+        to.setAttribute(attr.name, attr.value);
+      }
```

Comment:
As per the earlier paragraph in the docs, the correct snippet would be to set a `dom` option with the callback:

```suggestion
  dom: {
    onBeforeElUpdated(from, to) {
      for (const attr of from.attributes) {
        if (attr.name.startsWith("data-js-")) {
          to.setAttribute(attr.name, attr.value);
        }
      }
```

---

See examples in the official test suite:

- https://github.com/phoenixframework/phoenix_live_view/blob/6648f774354e308efc26d7e44e6e68d5a1b12fe1/assets/test/view_test.js#L918-L924
- https://github.com/phoenixframework/phoenix_live_view/blob/6648f774354e308efc26d7e44e6e68d5a1b12fe1/test/support/e2e/form_feedback.ex#L25-L58